src/clickChart.js

```diff
@@ -0,0 +1,78 @@
+import Chart from 'chart.js/auto';
+import { pageData } from './apiCalls';
+import { sortByHits } from './recipes';
+
+const makeRecipeClickChart = () => {
+    const chartContainer = document.querySelector('#chartContainer');
+    chartContainer.innerHTML = `<canvas id="clickChart"></canvas>`;
+    const sortedRecipes = sortByHits(pageData.allRecipes).slice(0, 9);
+    new Chart(document.querySelector('#clickChart'),
+        {
+            type: 'bar',
+            data: {
+                labels: sortedRecipes.map(recipe => splitLabelLength(recipe.name)),
+                fontColor: 'white',
+                datasets: [
+                    {
+                        label: 'Hits',
+                        data: sortedRecipes.map(recipe => recipe.hits),
+                        borderColor: '#fbbf83',
+                        backgroundColor: '#fbbf83',
+                    }
+                ]
+            },
+            options: {
+                indexAxis: 'y',
+                elements: {
+                    bar: {
+                        borderWidth: 2,
+                    }
+                },
+                responsive: true,
+                maintainAspectRatio: false,
+                plugins: {
+                    legend: {
+                        position: 'right',
+                    },
+                    title: {
+                        display: true,
+                        text: 'Recipes by Number of Clicks',
+                        font: {
+                            size: 40
+                        },
+                        color: 'white'
+                    }
+                },
+                scales: {
+                    x: {
+                        ticks:{
+                            display: true,
+                            color: 'white',
+                            stepSize: 1
+                        }
+                    },
+                    y: {
+                        ticks:{
+                            display: true,
+                            color: 'white',
+                            autoSkip: false
+                        }
+                    },
+                },
+                tooltips:{
+                    mode:'index'
+                }
+            }
+        }
+    );
+}
+
+const splitLabelLength = (recipeName) => {
+    if (recipeName.split(' ').length > 3) {
+        return recipeName.match(/\b[\w']+(?:[^\w\n]+[\w']+){0,2}\b/g);
```

Comment:
Do you think we could use something like this just for the readability?

`const firstString = "Creamy Coconut Yogurt Bowl with Chocolate Granola (Video)";

const splitCorrectly = string => {
  const firstArray = string.split(" ");
  const newArray = [];
  let tempString;
  const number = firstArray.length;
  
  for (var i = 0; i < number/3; i++) {
    if (!firstArray.length < 3) {
      tempString = firstArray.splice(0,3).join(" ");  
    } else {
      tempString = firstArray.splice(0, firstArray.length-1).join(" ")
    }
    
  newArray.push(tempString);
  }
  return newArray.join("\n")
}`